docs/timezone.md

```diff
@@ -45,6 +47,51 @@ tz1.getTransitions(now).next().done;  // => true
 tz2.getTransitions(now).next().done;  // => false
 ```
 
+## Temporal.TimeZone.from(thing: string | object) : Temporal.TimeZone
+
+**Parameters:**
+- `thing` (string or object): A `Temporal.TimeZone` object or a string from which to create a `Temporal.TimeZone`.
+
+**Returns:** a new `Temporal.TimeZone` object.
+
+This static method creates a new time zone from another value.
+If the value is a string, it can be:
+- a string that is accepted by `new Temporal.TimeZone()`;
+- a string in the ISO 8601 format including a time zone part, optionally extended with an IANA time zone name in square brackets appended to it;
+- or, a string with _only_ the time zone part of the ISO 8601 format, again extended with optional IANA name in brackets.
+Or, if the value is an object, it can be another `Temporal.TimeZone` object, which is returned directly.
+
+This function is often more convenient to use than `new Temporal.TimeZone()` because it handles a wider range of input.
+
+Usage examples:
+```javascript
+// IANA time zone names and UTC offsets
+tz = Temporal.TimeZone.from('UTC');
+tz = Temporal.TimeZone.from('Africa/Cairo');
+tz = Temporal.TimeZone.from('america/VANCOUVER');
+tz = Temporal.TimeZone.from('Asia/Katmandu');  // alias of Asia/Kathmandu
+tz = Temporal.TimeZone.from('-04:00');
+tz = Temporal.TimeZone.from('+645');
```

Comment:
This also is not valid ISO 8601 and therefore suspect. Why should Temporal.TimeZone.from accept it?

Reply:
Because we have decided that `Temporal.TimeZone.from` should accept a UTC offset string (regardless of whether we decide that a bare ISO 8601 time zone part is okay) and the polyfill currently considers that a UTC offset string. (The spec says nothing yet, since that's still marked TODO.) I think I agree that a legal UTC offset string should be the same as a legal UTC offset in an ISO 8601 time zone part, so I'll add another commit to this branch to bring that in line.